www/src/pages/foundations/colors.jsx

```diff
@@ -117,7 +117,7 @@ export default function ColorsPage({ data }) {
               marginBottom: '2rem',
             }}
           >
-            <h6>accents</h6>
+            <h2>accents</h2>
```

Comment:
Similar to above comment, let's keep this consistent with the other headings as an `<h2 className="h5">`